assets/airports/airportLoadList.js

```diff
@@ -46,6 +46,13 @@ window.AIRPORT_LOAD_LIST = (function() { // eslint-disable-line wrap-iife
      * @final
      */
     return [
+        {
+            icao: 'cyhz',
+            level: 'easy',
```

Comment:
Following the guidelines discussed in #655, `beginner` would suit the airport better since it has 10.752 acph